pol-core/pol/globals/uvars.cpp

```diff
@@ -185,7 +185,11 @@ GameState::~GameState()
   // or make sure that the globals get deconstructed before eg the flyweight string container
 }
 
-
+void GameState::initialize_range_from_config( u16 range )
+{
+  max_update_range_client = range;
```

Comment:
Since pol.cfg can be reread during runtime the new value should be only applied if it's greater then the current `max_update_range_client`

Reply:
At the moment this is called only on initial config load and not on reload ( https://github.com/polserver/polserver/blob/fb034e0d113acdbf8f74fb7e8fb44c9db0148fb5/pol-core/pol/polcfg.cpp#L55-L115 )

I'm not sure whether allowing reload is something we want to do because:
1) it needs to be in sync with client, I do not expect people changing this often
2) the dynamic view range doesn't seem to be properly implemented, so i'm bit worried about what it actually does https://github.com/polserver/polserver/blob/f45d12665d5b710f1643c96b44faaf1219c972ba/pol-core/pol/miscmsg.cpp#L475  https://github.com/polserver/polserver/blob/f45d12665d5b710f1643c96b44faaf1219c972ba/pol-core/pol/mobile/charactr.cpp#L4319

That being said I can change it if you confirm these aren't issues.